### ISSUE TITLE
run as service/tray icon

### ISSUE THREAD

**Issue by nanopone** [Kataiser/tf2-rich-presence#242]
> Run as a service (Abandoned because "launch TF2" button OP)
a button inside the app to launch tf2 is still inconvenient and doesn't replace the app being a background process. having the presence launch on startup and run in the background as a service or as a system tray icon is a lot more convenient. this requires no manual opening or closing of the presence, just launching tf2 like normal, which is a hell of a lot more seemless

**Comment by Kataiser** [comment]
Yeah but I still need the game to launch with -condebug. Previously I had to ask users to do that, and then detect if you didn't (which does still happen if you launch the game through Steam), but the launch button enforces that which is awesome. I do wish the experience was more seamless, but I think the current setup is the best solution.

**Comment by nanopone** [comment]
adding -condebug to your launch options then never thinking about it ever again is significantly less effort than having to open a seperate application. i still automatically open tf2 from steam like i have done for the past 10 years and then i have to close it again just to use rich presence. it often isn't worth the effort.

**Comment by starpotion** [comment]
If you want a _slightly_ more convenient way to streamline the process I threw `C:\Users\YOURUSERNAME\AppData\Local\Programs\TF2 Rich Presence\TF2 Rich Presence.bat %command%` into the advanced launch options. Program opens once you click play in Steam and then it's just another click to launch TF2 proper.

**Comment by RalphORama** [comment]
One solution could be an "auto launch TF2 when opened" option. Either a command line flag or a checkbox in settings. That way you could add a shortcut to TF2RP in Steam. Clicking play would open TF2RP, TF2RP would open TF2. About as seamless as you could ask for

**Comment by Kataiser** [comment]
Oh true probably

**Comment by Kataiser** [comment]
@RalphORama Done: https://github.com/Kataiser/tf2-rich-presence/commit/3354d7146282981a62121bef10c0a5fb7d0dd8e0
